src/components/app/details/appConfig/AppConfig.tsx

```diff
@@ -332,17 +332,17 @@ export default function AppConfig({ appName, isJobView, filteredEnvIds }: AppCon
             if (state.canDeleteApp) {
                 return (
                     <DeleteDialog
-                        title={`Delete '${appName}'?`}
+                        title={`Delete job '${appName}'?`}
```

Comment:
Did you verify this change? I think it will impact the app flow. please verify and if required add the `isJobView` check